src/panels/developer-tools/logs/error-log-card.ts

```diff
@@ -35,7 +35,9 @@ class ErrorLogCard extends LitElement {
             `
           : html`
               <mwc-button raised @click=${this._refreshErrorLog}>
-                Load Full Home Assistant Log
+                ${this.hass.localize(
+                  "ui.panel.developer-tools.tabs.logs.load_log"
```

Comment:
```suggestion
                  "ui.panel.developer-tools.tabs.logs.load_full_log"
```

Reply:
Addressed in https://github.com/home-assistant/home-assistant-polymer/pull/4046/commits/fc577b40f43570812f8c60356fbdf1241136426c :)